<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowserPage</class>
 <widget class="QWidget" name="BrowserPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>501</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="ledCurrentDir"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbtHome">
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/wrx.qrc">
         <normaloff>:/img/btnHome.png</normaloff>:/img/btnHome.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtDirUp">
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/wrx.qrc">
         <normaloff>:/img/btnUp.png</normaloff>:/img/btnUp.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtPrev">
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/wrx.qrc">
         <normaloff>:/img/btnLeft.png</normaloff>:/img/btnLeft.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtNext">
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/wrx.qrc">
         <normaloff>:/img/btnRight.png</normaloff>:/img/btnRight.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtReload">
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/wrx.qrc">
         <normaloff>:/img/btnReload.png</normaloff>:/img/btnReload.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtShow">
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/wrx.qrc">
         <normaloff>:/img/btnOpen.png</normaloff>:/img/btnOpen.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stwBrowser">
     <widget class="QWidget" name="stwBrowserPageDir">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="lwgFiles"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stwBrowserPageFax">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="stwBrowserPageFaxWdgPicture" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="QScrollBar" name="stwBrowserPageFaxScbVert">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QScrollBar" name="stwBrowserPageFaxScbHoriz">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stwBrowserPageText">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/wrx.qrc"/>
  <include location="resources/wrx.qrc"/>
 </resources>
 <connections/>
</ui>
